### DISCUSSION TITLE
SshError

### DISCUSSION THREAD

**Discussion by soffoalbert** [ubicloud/ubicloud#1367] [Q&A] [answered]
Please help.
 
While setting my ubicloud on Hetzner. 

I am getting the following error:

15:04:07 respirate.1 | {"exception":{"message":"command exited with an error: sudo host/bin/setup-nftables.rb \\[\\]","class":"Sshable::SshError","backtrace":["/app/model/sshable.rb:94:in `cmd'","/app/prog/setup_nftables.rb:8:in `start'","/app/model/strand.rb:109:in `public_send'","/app/model/strand.rb:109:in `block (2 levels) in unsynchronized_run'","/app/lib/sem_snap.rb:17:in `block in use'","/app/lib/sem_snap.rb:22:in `use'","/app/lib/sem_snap.rb:16:in `use'","/app/model/strand.rb:106:in `block in unsynchronized_run'","/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/transactions.rb:264:in `_transaction'","/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/transactions.rb:239:in `block in transaction'","/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/connection_pool/threaded.rb:92:in `hold'","/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/connecting.rb:293:in `synchronize'","/usr/local/bundle/gems/sequel-5.78.0/lib/sequel/database/transactions.rb:197:in `transaction'","/app/model/strand.rb:105:in `unsynchronized_run'","/app/model/strand.rb:152:in `block (2 levels) in run'","/app/model/strand.rb:151:in `loop'","/app/model/strand.rb:151:in `block in run'","/app/model/strand.rb:36:in `take_lease_and_reload'","/app/model/strand.rb:150:in `run'","/app/scheduling/dispatcher.rb:48:in `block in start_strand'"],"cause":"nil"},"message":"exception terminates thread","time":"2024-03-18 15:04:07 +0000","thread":"st06753zysxy1p1ztrc8qr93sm"} 

i set the ssh key in my vm s authroizedkeys is this correct? or should it be in that of the docker container?

Thanks in advance.

**Comment by furkansahin** [comment, marked as answer]
Hello @soffoalbert ,
Thank you for asking this. The issue happens when you have a hetzner server without an additional subnet. If you have added an additional subnet, this should not have happened. Also, keep in mind, if you are planning to use IPv4 addresses, you should get an additional subnet from Hetzner. In any case, to fix this issue, I have created this PR https://github.com/ubicloud/ubicloud/pull/1386. 
Best,
Furkan

**Comment by furkansahin** [comment]
The PR is merged in.
